src/app/subscription/page.tsx

```
@@ -0,0 +1,577 @@
+            });
+
+            const payhereResult = await payhereResponse.json();
+
+            if (!payhereResponse.ok) {
+                throw new Error(
+                    payhereResult.message ||
+                        `Failed to generate PayHere link: ${payhereResponse.statusText}`
+                );
+            }
+
+            if (payhereResult && payhereResult.link) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)